src/utils/worker/resolve.ts

```diff
@@ -1,8 +1,32 @@
+import { prisma } from '../prisma';
+
 import * as emailTemplates from './mail/templates';
 import { sendMail } from './mail';
 import { JobDataMap } from './create';
+import { sheep } from './sheep';
 
 export const email = async ({ template, data }: JobDataMap['email']) => {
     const renderedTemplate = await emailTemplates[template](data);
     return sendMail(renderedTemplate);
 };
+
+export const goalPing = async () => {
+    const targetDate = new Date();
+
+    targetDate.setMonth(targetDate.getMonth() - 1);
+
+    const goals = await prisma.goal.findMany({
+        where: {
+            updatedAt: {
+                lte: targetDate,
+            },
+        },
+    });
+
+    sheep.createComments(
```

Comment:
I'm not sure it's nice idea to create all comments in same time. This is can be a perf issue. I think it would be better to create jobs which creates comments with interval.